src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -312,7 +312,7 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
                         key => repository.ValueAsString(key).Value);
             }
 
-            project.BuildTargetFrameworksAndConfigurations(rawProject);
+            project.BuildTargetFrameworksAndConfigurations(rawProject, diagnostics);
```

Comment:
This `diagnostics` is for output purpose. The check will be done by caller of this method.